src/cpu/aarch64/acl_softmax.cpp

```diff
@@ -111,8 +107,8 @@ status_t acl_softmax_fwd_t::pd_t::init(engine_t *engine) {
         // A rough empirical heuristic, see comment above
         // The only difference here is that ACL does a reorder, and so
         // is considerably better
-        double acl_ref_performance_diff = calculate_performance_diff(
-                outer_size_, axis_size_, threads, -0.01);
+        double acl_ref_performance_diff
+                = calculate_performance_diff(-0.01 * inner_size_);
```

Comment:
IIUC, this is the only impactful change (change from `-0.01` to `-0.01 * inner_size_`), is that correct?

Reply:
Yes, it's the only line of code.